src/main/java/gov/cabinetoffice/gap/adminbackend/controllers/ApplicationFormController.java

```diff
@@ -118,6 +131,46 @@ public ResponseEntity deleteApplicationForm(@PathVariable @NotNull Integer appli
 
     }
 
+    @PostMapping("/lambda/{grantAdvertId}")
```

Comment:
POST `/application-forms/lambda/{grantAdvertId}`

Doesn't describe what the request mapping is doing

Reply:
Switched to DELETE `/application-forms/lambda/{grantAdvertId}/application`